### DISCUSSION TITLE
toggling question

### DISCUSSION THREAD

**Discussion by yushisalvador** [dowjones/react-dropdown-tree-select#614] [General]
Hi, I was wondering if it's possible to only allow one leaf node open at a time?

Desired:
I have a parent node with 3 options/dropdowns, When dropdown 1 is opened, and try to open dropdown 2, dropdown 1 is closed automatically.

**Comment by mrchief** [comment]
I'm not sure I follow. Are you talking about nodes within the same dropdown or different dropdowns altogether?

**Reply by yushisalvador** [reply]
I'm talking about one parent dropdown with multiple children dropdowns. 

For instance I have a dropdown called location, when i open location, different continents shows up. those continents are their own dropdowns with countries. When I open dropdown africa, then click on dropdown europe, dropdown africa is closed and europe is opened.

**Reply by mrchief** [reply]
Right - that is something very custom for your use case. One instance knows nothing about the other. There are roundabout ways of keeping the dropdowns open (hint: css overrides) but it's all very hacky UX. You could fork this and tweak for your own needs or you could go for a different UX.
